### DISCUSSION TITLE
Newbie Question Construction Timelapse

### DISCUSSION THREAD

**Discussion by gavin1f4** [gopro/labs#1040] [General]
Hi,

I am new and trying to learn to make custom qr codes for construction time-lapse. Is there a write up or a guide to help me learn to create custom features? If I am understanding correctly its just a QR code of a text string of the command codes to make a feature? Is there a way to build off of a feature that is already created. For instance there is a feature Daily Time-lapse Video and Optional Upload which I would like to do but with photos instead of videos. 

Thanks

**Comment by dnewman-gpsw** [comment]
You have to learn how the current feature work.  Docs are here: https://gopro.github.io/labs/control/tech/

Then you can rebuild and customize with https://gopro.github.io/labs/build/

However daily photo timelapse with upload is not likely to be very reliable. It will require a good and fast WiFi Network.  Video more efficient and therefore more reliable on a wider range of networks.

**Reply by gavin1f4** [reply]
Thanks for the quick supply. I have read through those still figuring out how to use the builder tool. Is it just the upload part that isn’t reliable? That part isn’t a must have. The daily photo between certain times and a certain interval is important. Has that part been reliable with the camera turning off and back on for each photo? It would be plugged in to power.

**Reply by dnewman-gpsw** [reply]
HERO12 is 100% reliable for time scripted captures, I've had a HERO12 running a timelapse all year. HERO11 is pretty good, I would check it once a week.  All older models are good for average of 300-400 images for both something random prevents the script from continuing.
